lib/config/presets/internal/config.ts

```diff
@@ -15,6 +15,20 @@ export const presets: Record<string, Preset> = {
       'workarounds:all',
     ],
   },
+  'best-practices': {
+    description: 'Preset with best practices from the Renovate maintainers.',
+    extends: [
+      'config:base',
+      'docker:pinDigests',
+      'helpers:pinGitHubActionDigests',
+    ],
+    packageRules: [
+      {
+        matchDepTypes: ['devDependencies'],
+        rangeStrategy: 'pin',
+      },
+    ],
```

Comment:
Let's make this into a preset if one doesn't already exist (`:pinDevDependencies`)